src/App.tsx

```diff
@@ -1,27 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+// import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 // or
 // import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  async function handlerLoadGood(f: () => Promise<Good[]>) {
+    const result = await f();
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+    setGoods(result);
+  }
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  // eslint-disable-next-line no-console
+  console.log([goods]);
```

Comment:
Remove the console.log statement as it is not needed and can cause performance issues.